corehq/apps/custom_data_fields/static/custom_data_fields/js/custom_data_fields.js

```diff
@@ -39,6 +39,12 @@ hqDefine('custom_data_fields/js/custom_data_fields', [
             return Choice(choice);
         }));
         self.validationMode = ko.observable(options.choices.length ? 'choice' : 'regex');
```

Comment:
Should this line be removed since the line below it appears to be overwriting the same name?

Reply:
Yikes. This was an incomplete fix, thanks